pkg/yurttunnel/agent/cmd.go

```diff
@@ -79,8 +78,8 @@ func NewYurttunnelAgentCommand(stopCh <-chan struct{}) *cobra.Command {
 		"The identifiers of the agent, which will be used by the server when choosing agent.")
 
 	// add klog flags as the global flagsets
-	klog.InitFlags(nil)
-	flags.AddGoFlagSet(flag.CommandLine)
+	//klog.InitFlags(nil)
```

Comment:
How about just removing those lines instead of commenting them out.

Reply:
fixed